packages/components/e2e/component-objects/src/main/java/org/talend/component/list/large/Large.java

```diff
@@ -0,0 +1,131 @@
+package org.talend.component.list.large;
+
+import static java.util.stream.Collectors.toList;
+
+import java.util.List;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.NotFoundException;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.talend.component.Component;
+
+/**
+ * A List is used to easy access to WebElements of the react-talend-component List component - Large.
+ */
+public class Large extends Component {
+
+    private static final String NAME = "Large";
+
+    private static final String LIST_LARGE_SELECTOR = ".ReactVirtualized__List";
+
+    private static final String TABLE_ITEM_SELECTOR = ".tc-list-large";
+
+    /**
+     * Constructor.
+     *
+     * @param driver Selenium WebDriver
+     */
+    public Large(final WebDriver driver) {
+        this(driver, null);
+    }
+
+    /**
+     * Constructor with table id
+     *
+     * @param driver Selenium WebDriver
+     */
+    public Large(final WebDriver driver, final String id) {
+        super(driver, NAME, String.format(LIST_LARGE_SELECTOR, id != null ? "#" + id : ""));
+    }
+
+    /**
+     * Get all rendered items, represented by a row in the table.
+     *
+     * @return The list of Items
+     */
+    public List<ItemLarge> getDisplayedItems() {
+        return this.getElement() //
+                .findElements(By.cssSelector(TABLE_ITEM_SELECTOR)) //
+                .stream() //
+                .map(webElement -> new ItemLarge(driver, webElement)) //
+                .collect(toList());
+    }
+
+    /**
+     * Scroll to top
+     */
+    public void scrollToTop() {
```

Comment:
it might be a good thing to mutualize the different functions (use an argument for exemple)
scrollToTop
canScrollDown
getItem
hasItem

because the differences are really minimal with the classe Table